assets/css/styleguide.scss

```diff
@@ -1020,6 +1035,13 @@ $font-light: 300;
       font-size: rem(15px);
       line-height: 1.7;
```

Comment:
@angel I didn't realize we were using 1.7 for our default style so I added it as a variable in _variables.scss + _default.scss in 3f26be6.